app.js

```diff
@@ -0,0 +1,91 @@
+import checkResult from './battlefunctions.js';
+let userAttack = document.getElementById('attackbutton').value;
+const randomGodzillaattack = Math.random();
+let godzillaAttack = 'ROCK';
+// initialize/fetch DOM elements
+let winner = 0;
+const attackButton = document.getElementById('attackbutton');
+let userScore = 0;
+let godzillaScore = 0;
+evaluateGodzillaStatusAndPlanAttack();
+
+
+// set some initial state
+
+
+
+// define DOM utility function
+
+
+
+
```

Comment:
Go ahead and delete some of the unnecessary whitespace/comments as you go, so that when you submit it's cleaner code :)